frontend/ui/uimanager.lua

```diff
@@ -1841,6 +1847,12 @@ function UIManager:_standbyTransition()
     self._prev_prevent_standby_count = self._prevent_standby_count
 end
 
+-- Used by a PM transition event handler to request an early return from input polling (value in s).
+-- NOTE: We can't re-use setInputTimeout to avoid interactions with ZMQ...
```

Comment:
ZMQ ? What's that ? :)
I remember seeing ZeroMQ in bits of code - but thought this was disabled (unless I'm confusing it with TurboSomething...).
Anyway, never had to meet this thing :) What is it used for and why ? Couldn't we don't go with it ?
(Thanks for a little 3 sentences primer to keep me in the know :)

Reply:
Zmq is about communicating between different processes. I think it's used in a (deprecated?) plugin for document sharing between KOReader instances.